src/app/public/modules/dropdown/dropdown-menu.component.ts

```diff
@@ -284,4 +282,130 @@ export class SkyDropdownMenuComponent implements AfterContentInit, OnDestroy {
       return (i === index);
     });
   }
+
+  private sendMessage(type: SkyDropdownMessageType): void {
+    this.dropdownComponent.messageStream.next({ type });
+  }
+
+  private addEventListeners(): void {
+    const dropdownMenuElement = this.elementRef.nativeElement;
+
+    Observable
+      .fromEvent(dropdownMenuElement, 'click')
+      .takeUntil(this.ngUnsubscribe)
+      .subscribe((event: MouseEvent) => {
+        const selectedItem = this.menuItems.find((item: SkyDropdownItemComponent, i: number) => {
+          const found = (item.elementRef.nativeElement.contains(event.target));
+
+          if (found) {
+            this.menuIndex = i;
+            this.menuChanges.next({
+              activeIndex: this.menuIndex
+            });
+          }
+
+          return found;
+        });
+
+        /* istanbul ignore else */
+        if (selectedItem) {
+          this.menuChanges.next({
+            selectedItem
+          });
+        }
+      });
+
+    Observable
+      .fromEvent(dropdownMenuElement, 'keyup')
```

Comment:
I noticed that when pressing 'enter' on any menu items, it will activate that item, but then move focus to the top of the DOM. With the current version, pressing `enter` will still keep focus on the dropdown. Should we keep this behavior?

Reply:
Good catch!